ansible_collections/arista/avd/roles/eos_designs/python_modules/l3_edge/router_isis.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
+
+from .utils import UtilsMixin
+
+
+class RouterIsisMixin(UtilsMixin):
+    """
+    Mixin Class used to generate structured config for one key.
+    Class should only be used as Mixin to a AvdStructuredConfig class
+    """
+
+    @cached_property
+    def router_isis(self) -> dict | None:
+        """
+        Return structured config for router_isis
+        """
+
+        if not self._underlay_isis:
+            return None
+
+        no_passive_interfaces = [p2p_link["data"]["interface"] for p2p_link in self._filtered_p2p_links if p2p_link.get("include_in_underlay_protocol") is True]
+        if no_passive_interfaces:
```

Comment:
This no_passive_interfaces data model is actually not supported by eos_cli_config_gen under router isis (you'll notice the rendered cli config doesn't do anything with this), i think this was just copy pasted from the ospf template at some point so i don't think we need this... Also the passive/no passive commands in EOS under isis doesn't actually configure anything under the isis process, it's only a shortcut to set passive/no passive on the interface configuration level.

Reply:
Good catch. I have removed router_isis from this PR, since this was the only key set there.